<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfitExplorerPage</class>
 <widget class="QWidget" name="ProfitExplorerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true">#checkBox { color: #4CFF00; }</string>
         </property>
         <property name="text">
          <string>Auto Refresh</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="networkCheckBox">
         <property name="styleSheet">
          <string notr="true">color: #cecece;</string>
         </property>
         <property name="text">
          <string>Show Network</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">#label { color: #ffffff; }</string>
         </property>
         <property name="text">
          <string>Blocks To Go Back:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recomputeButton">
         <property name="text">
          <string>Recompute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCustomPlot" name="difficultyPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#difficultyPlot { background-color: #202020; }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#customPlot { background-color: #202020; }</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color: #cccccc; background-color: #000000;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Staking:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stakingLabel">
         <property name="styleSheet">
          <string notr="true">color: #4CFF00;background-color:#000000;</string>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: #cccccc; background-color: #000000;</string>
         </property>
         <property name="text">
          <string>Difficulty:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="difficultyLabel">
         <property name="styleSheet">
          <string notr="true">color: #4CFF00;background-color:#000000;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color: #cccccc; background-color: #000000;</string>
         </property>
         <property name="text">
          <string>Weight:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weightLabel">
         <property name="styleSheet">
          <string notr="true">color: #4CFF00;background-color:#000000;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: #cccccc; background-color: #000000;</string>
         </property>
         <property name="text">
          <string>Network Weight:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="netWeightLabel">
         <property name="styleSheet">
          <string notr="true">color: #4CFF00;background-color:#000000;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">color: #cccccc; background-color: #000000;</string>
         </property>
         <property name="text">
          <string>Time to stake:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeToStakeLabel">
         <property name="styleSheet">
          <string notr="true">color: #4CFF00;background-color:#000000;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
